packer/ansible/roles/aggregate/files/motd/10-services

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+PROPS_FILE=/var/lib/tomcat8/webapps/ROOT/WEB-INF/classes/security.properties
+
+rawHostname=$(grep hostname ${PROPS_FILE} | awk '{split($0,a,"="); print a[2]}')
+hostname=${rawHostname:-localhost}
```

Comment:
It would be really nice to print the IP of enp0S3 or whatever the non-loop back IP address was. That's the IP that users who aren't NATed would use. Users shouldn't have to run `ip address` to figure that out.

Reply:
Also, I think we should move this to /etc/issue so that this information prints without users having to login.

Reply:
This is the second version of this script. I'll rescue some code from the previous version which does just that.

We show this message now:
```
Open a browser and use one of these URLs to access ODK Aggregate
http://cocotero.com:10080
https://cocotero.com:10443

This VM uses these IP addresses:
10.0.2.15
```

The nice thing about MOTD is that it has a specific system to support dynamic MOTDs.

`issue`, while it lets you insert some basic stuff, it's not so nice to make it dynamic like we want. We can make a script that writes the contents of `/etc/issue`, but I don't see an easy way to detect networking changes and hook the script to them to update the `issue` file whenever there's a change.

I've made some tests and tried placing the script on `/etc/network/if-up.d` and `/etc/network/if-post-down.d` but it doesn't detect when I change the VM's network interface on a live machine.

What I've done is to add the script to `/etc/rc.local` to have it change the `issue` message while booting.

Then `aggregate-config` runs the update script at the end. It's not perfect but I think that it could be enough... what do you think?